local.properties

```diff
@@ -5,4 +5,5 @@
 # For customization when using a Version Control System, please read the
 # header note.
 #Thu Sep 08 11:41:29 CEST 2022
-ndk.dir=/usr/local/lib/android/sdk/ndk/25.1.8937393
+#ndk.dir=/usr/local/lib/android/sdk/ndk/25.1.8937393
```

Comment:
we need this path for the pipeline

Reply:
The GitHub action fails without those changes,
For the pipeline, It's passing with change.